### PR TITLE
Feature/add js and build process

### PR DESCRIPTION
**Pull request by menno-ll**
## Issue link

https://app.asana.com/0/1195088574671066/1198951512972459/f

## Description

Adds CSS support to plugin
Adds JS support to plugin
Adds build script to github actions
Adds JS to trigger cart button and buy buttons

## How Has This Been Tested?

The build: i created [a release](https://github.com/level-level/ll-voordemensen/releases/tag/test-release-script-4)

The buttons: clicking on it

## Screenshots (if appropriate):

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          cd wp-content/plugins/ll-voordemensen/
+          npm run build
+      - name: Setup Satis authentication
+        run: |
+          cd wp-content/plugins/ll-voordemensen/
+          composer config http-basic.${{ secrets.SATIS_DOMAIN }} "${{ secrets.SATIS_USERNAME }}" "${{ secrets.SATIS_PASSWORD }}"
+        env:
+          SATIS_DOMAIN: ${{ secrets.SATIS_DOMAIN }}
+          SATIS_USERNAME: ${{ secrets.SATIS_USERNAME }}
+          SATIS_PASSWORD: ${{ secrets.SATIS_PASSWORD }}
```
Comment (NielsdeBlaauw) [inline .github/workflows/build.yml:63]: Ik hoop dat dit niet nodig is. We gebruiken denk ik alleen `level-level/coding-standards` als prive repository? Kunnen we voor dit project niet beter de openbare WP Coding Standards aanhouden? Dan kan dit deel er van af.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      - name: Install Composer dependencies (including dev)
+        run: |
+          cd wp-content/plugins/ll-voordemensen/
+          composer install --prefer-dist --no-suggest --ignore-platform-reqs
```
Comment (NielsdeBlaauw) [inline .github/workflows/build.yml:67]: Volgens mij hebben we de dev dependencies hier niet nodig? Want de make pot is vanuit WP cli zelf?

**Review by NielsdeBlaauw — APPROVED** [review]
Volgens mij werkt het verder. Check wel even de comments.

File: .github/workflows/build.yml (hunk 1 above)
Comment (menno-ll) [reply .github/workflows/build.yml]: https://github.com/level-level/ll-voordemensen/pull/16/commits/417a9aebc6576441a428910ba86efe834f58d021

File: .github/workflows/build.yml (hunk 2 above)
Comment (menno-ll) [reply .github/workflows/build.yml]: https://github.com/level-level/ll-voordemensen/pull/16/commits/f731d3a1851200ab99b343b454c0a54dc2447cd9

**Review by NielsdeBlaauw — APPROVED** [review]

**Pull request merged**